build/yamls/base/antctl.yml

```diff
@@ -47,3 +56,9 @@ subjects:
     # ServiceAccount token.
     name: antrea-controller
     namespace: kube-system
+  - kind: ServiceAccount
+    # This is to allow antctl to be executed inside the Antrea agent Pod
+    # and authenticate with agent API server using the Controller's
+    # ServiceAccount token.
+    name: antrea-agent
```

Comment:
Confused. Why we create antrea-agent SA here? It is created in agent-rbac.yml already.

Did not understand your comments either - antctl in Agent Pod uses Controller SA token? Guess you should check and update.

Reply:
Service account is defined withing agent-rbac.yml while this is the cluster role binding with antctl which is needed to both antrea-controller and antrea-agent.
It doesn't work without it.

Reply:
Ok. My mistake. Sorry.